Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM  adoptopenjdk/openjdk11:alpine AS build
+WORKDIR /app
+COPY . ./
+RUN ./gradlew clean build
+
+FROM adoptopenjdk/openjdk11:alpine AS unit-test
+WORKDIR /app
+COPY --from=build /app/src .
+RUN ./gradlew clean check
+
+FROM adoptopenjdk/openjdk11:alpine AS beacons-service
```

Comment:
Ended up using [this](https://github.com/aadilp/spring-podman/blob/f6d8bfddb3a0804cd027dde18785e3a26c44fd65/Dockerfile) as inspo and using a different base image for building ([gradle one which is official](https://hub.docker.com/_/gradle), so good?) because I kept getting an error I couldn't figure out with the adoptopenjdk.

![image](https://user-images.githubusercontent.com/32230328/105335116-902a9600-5bcf-11eb-99f8-192290831e62.png)